spring-rabbit/src/main/java/org/springframework/amqp/rabbit/listener/DirectMessageListenerContainer.java

```diff
@@ -196,6 +202,29 @@ public final void setMissingQueuesFatal(boolean missingQueuesFatal) {
 		super.setMissingQueuesFatal(missingQueuesFatal);
 	}
 
+	/**
+	 * Set the number of messages to receive before acknowledging (success).
+	 * A failed message will short-circuit this counter.
+	 * @param messagesPerAck the number of messages.
+	 * @see DirectMessageListenerContainer#setAckTimeout(long)
```

Comment:
We don't need the class prefix if we refer to the API from the same class.
Will fix one merge if only concern